src/content/docs/en/reference/api-reference.mdx

```diff
@@ -426,6 +426,25 @@ And would render HTML like this:
 </ul>
 ```
 
+
+### `Astro.locals`
+
+`Astro.locals` is an API available via [Astro middleware](/en/guides/middleware/). It's an object that can contain any arbitrary information.
```

Comment:
```suggestion
`Astro.locals` is an object containing any values from the [`context.locals`](#contextlocals) object from a middleware. Use this to access data returned by middleware in your `.astro` files.
```
OK, PLOT TWIST!  :sweat_smile: 

So I looked at the definition for `Astro.props` (because, this is like props for middleware). What do you think about something like this instead?

Reply:
That works very well! Thank you!